pages/Documents/MessagingChannels/AppleBusinessChatSDK/overview.md

```diff
@@ -4,23 +4,23 @@ redirect_from:
 Keywords:
 sitesection: Documents
 categoryname: "Messaging Channels"
-documentname: Apple Business Chat SDK
-permalink: apple-business-chat-sdk-overview.html
+documentname: Apple Messages for Business SDK
```

Comment:
Dto., `documentname` needs updating in documentsupdated.yaml.

Also, please add a redirect.

Reply:
reverted -- will address page names and redirects separately